components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-finish-local-ajaxprocessor.jsp

```diff
@@ -111,6 +111,14 @@
         }
         property.setValue(samlMetadataSigningEnabled);
         propertyList.add(property);
+        property = new Property();
+        property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.SAML_METADATA_AUTHN_REQUESTS_SIGNING_ENABLED);
+        String samlAuthnRequestsSigningEnabled = "false";
+        if (StringUtils.containsIgnoreCase(request.getParameter("samlAuthnRequestsSigningEnabled"), "on")) {
```

Comment:
Can we use the defined constant(SAML_METADATA_AUTHN_REQUESTS_SIGNING_ENABLED) here

Reply:
other request params are obtained using the same way. Hence you followed the same for consistency